### ISSUE TITLE
Email reminders for shows with incomplete information?

### ISSUE THREAD

**Issue by philosophicles** [camdram/camdram#787]
If a show listing is approved (publicly visible) but has any obviously missing information (venue / dates to be confirmed; no funding society listed), we could send automated reminder emails to the show admins (and any upstream admins that can be identified, if any of venue or soc are listed) to encourage this information to be filled in. 

To avoid being really annoying, if performance dates are known, then reminders for other fields could be limited to 1 month before, 1 week before, and 1 week after the first performance. If dates aren't known, it could be 1 week after listing created, 1 month after created, and 3 months after created. 

For funding society, this might require us adding an explicit "N/A" type option to the interface -- occasionally, shows are just self-funded with no society wanting to be mentioned. 

Similarly, we might need to figure out how to handle films with no real venue before we can implement this for missing venues.

**Comment by GKFX** [comment]
Good idea. We could also include ticket information in the list of information to provide.

That said, we should probably be looking to reduce the number of emails we’re sending now before sending more (e.g. aggregating notifications into fewer emails). As people do need to come back to show pages repeatedly to add roles, putting the reminders on the show page itself might be sufficient.

**Comment by philosophicles** [comment]
That's true - changing the scope of this to just "reminders of some sort" :) 

I think _some_ email reminders could still have a place here, because some show listings get created then more or less completely forgotten about - no description, no roles, etc, and the creator forgets to come back to Camdram at all. (Complete lack of these things would also merit a prompt/reminder!) 

But reminders in boxes on the show page (for logged-in show admins) would probably cover 80% or more of the cases. 

Email reminders could even perhaps only target show admins who haven't logged in in a while, _and_ where show info is missing.
